.github/workflows/publish.yml

```
@@ -1,29 +1,61 @@
-name: publish_docs
+name: "publish_docs"
 
 on:
   push:
     branches:
-    - main
+    - "main"
+    - "mimir-*"
+    tags:
+    - "v[0-9]+.[0-9]+.[0-9]+"
     paths:
-    - 'docs/sources/**'
+    - "docs/sources/**"
 
 jobs:
+  test:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Check out code"
+        uses: "actions/checkout@v3"
+      - name: "Build Website"
```

Can we use "Build site" instead, if this is just building a Hugo site?

---

Alternatively, "Build technical documentation"?

---

I believe this is actually building the whole website.
It should be sentence cased though for sure.
```suggestion
      - name: "Build website"
```